content/opt/setup/61-icingaweb2-director

```diff
@@ -16,6 +16,7 @@ mysql_wait \
 
 if evaluate_boolean "${ICINGA2_FEATURE_DIRECTOR}"; then
 
+  ln -fs /usr/share/icingaweb2/modules/director /etc/icingaweb2/enabledModules/
 	icingacli module enable director
```

Comment:
these two lines are doing the same thing, or are there differences?

Reply:
You're right, I removed one.